packages/client/wasm.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  ...require('.prisma/client/wasm'),
+}
```

Comment:
Nit, but is there a reasone why we don't do:
```suggestion
module.exports = require('.prisma/client/wasm');
```

just to save on a spread?

Reply:
I think this was required by some framework for some reason

Reply:
Found it: https://github.com/prisma/prisma/pull/12907

Reply:
We should add a comment for the future here then.